insights/client/__init__.py

```diff
@@ -344,11 +344,11 @@ def collect(self, **kwargs):
             logger.debug('Check timestamp is True.')
             cached_results = self._cached_results()
             if cached_results:
-                logger.info("Using cached collection: %s", cached_results)
+                logger.debug("Using cached collection: %s", cached_results)
                 # if to_stdout is flagged then do sys.stdout
                 if config["to_stdout"]:
-                    logger.info("Flag --to-stdout used, using system out.")
-                    logger.info(sys.stdout)
+                    logger.debug("Flag --to-stdout used, using system out.")
+                    logger.debug(sys.stdout)
```

Comment:
Shouldn't this use the `INFO: ` thing rather than `logger.debug`?

Reply:
No, this was just added in to test how the wrapper RPM was passing 'sys' to the Core to ensure we still had a correct file pointer for shutil to output the stdout for the .tar.gz file (when using --to-stdout).

Reply:
So should it be removed now?

Reply:
It hath been removed.